### PR TITLE
Multidriz model

### PR DESCRIPTION
**Pull request by stsci-hack**
This change adds the definition for a datamodel that supports multiple DrizProduct models in a manner similar to the MultiSlitModel datamodel.

### CODE REVIEW

File: jwst/datamodels/multiprod.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import, unicode_literals, division, print_function
+
+from . import model_base
+from .image import ImageModel
+
+
+__all__ = ['MultiSlitModel']
```
Comment (hbushouse) [inline jwst/datamodels/multiprod.py]: Forgot to update this line.

File: jwst/datamodels/multiprod.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    Parameters
+    ----------
+    init : any
+        Any of the initializers supported by `~jwst.datamodels.DataModel`.
+    """
+    schema_url = "multiproduct.schema.yaml"
+
+    def __init__(self, init=None, **kwargs):
+        if isinstance(init, ImageModel):
```
Comment (hbushouse) [inline jwst/datamodels/multiprod.py]: Shouldn't this be "DrizProductModel"? An ImageModel doesn't have the WHT and CON attributes that are referenced later.

**Comment by stsci-hack** [comment]
This PR should now be ready for review and inclusion into calwebb_spec2 updates by @jdavies-st after cleaning up the problems identified by Howard.

**Comment by hbushouse** [comment]
👍

**Pull request merged**